specifications/api/swagger.yaml

```diff
@@ -210,8 +210,8 @@ transactionId: &transactionId
   description: A unique identifier for this transaction
   type: string
   format: hex
-  maxLength: 64
-  minLength: 64
+  maxLength: 128
+  minLength: 128
```

Comment:
According to the comment https://github.com/input-output-hk/cardano-wallet/blob/5c7d690e5d994c931e5b437c5bdbaf222f6dc675/src/Cardano/Wallet/Binary.hs#L569 , `Hash "Tx"` is a 64 byte string, which translates to 128 long hex encoded string (because byte is encoded as 2 char string).